doc/api/os.md

```diff
@@ -33,205 +44,294 @@ defined are described in [OS Constants][].
-       nice: 0,
-       sys: 26980,
-       idle: 1071569080,
-       irq: 0 } },
-  { model: 'Intel(R) Core(TM) i7 CPU         860  @ 2.80GHz',
+    times: {
+      user: 306960,
+      nice: 0,
+      sys: 26980,
+      idle: 1071569080,
+      irq: 0
+    }
+  },
+  {
+    model: 'Intel(R) Core(TM) i7 CPU         860  @ 2.80GHz',
     speed: 2926,
-    times:
-     { user: 248450,
-       nice: 0,
-       sys: 21750,
-       idle: 1070919370,
-       irq: 0 } },
-  { model: 'Intel(R) Core(TM) i7 CPU         860  @ 2.80GHz',
+    times: {
+      user: 248450,
+      nice: 0,
+      sys: 21750,
+      idle: 1070919370,
+      irq: 0
+    }
+  },
+  {
+    model: 'Intel(R) Core(TM) i7 CPU         860  @ 2.80GHz',
     speed: 2926,
-    times:
-     { user: 256880,
-       nice: 0,
-       sys: 19430,
-       idle: 1070905480,
-       irq: 20 } },
-  { model: 'Intel(R) Core(TM) i7 CPU         860  @ 2.80GHz',
+    times: {
+      user: 256880,
+      nice: 0,
+      sys: 19430,
+      idle: 1070905480,
+      irq: 20
+    }
+  },
+  {
+    model: 'Intel(R) Core(TM) i7 CPU         860  @ 2.80GHz',
     speed: 2926,
-    times:
-     { user: 511580,
-       nice: 20,
-       sys: 40900,
-       idle: 1070842510,
-       irq: 0 } },
-  { model: 'Intel(R) Core(TM) i7 CPU         860  @ 2.80GHz',
+    times: {
+      user: 511580,
+      nice: 20,
+      sys: 40900,
+      idle: 1070842510,
+      irq: 0
+    }
+  },
+  {
+    model: 'Intel(R) Core(TM) i7 CPU         860  @ 2.80GHz',
     speed: 2926,
-    times:
-     { user: 291660,
-       nice: 0,
-       sys: 34360,
-       idle: 1070888000,
-       irq: 10 } },
-  { model: 'Intel(R) Core(TM) i7 CPU         860  @ 2.80GHz',
+    times: {
+      user: 291660,
+      nice: 0,
+      sys: 34360,
+      idle: 1070888000,
+      irq: 10
+    }
+  },
+  {
+    model: 'Intel(R) Core(TM) i7 CPU         860  @ 2.80GHz',
     speed: 2926,
-    times:
-     { user: 308260,
-       nice: 0,
-       sys: 55410,
-       idle: 1071129970,
-       irq: 880 } },
-  { model: 'Intel(R) Core(TM) i7 CPU         860  @ 2.80GHz',
+    times: {
+      user: 308260,
+      nice: 0,
+      sys: 55410,
+      idle: 1071129970,
+      irq: 880
+    }
+  },
+  {
+    model: 'Intel(R) Core(TM) i7 CPU         860  @ 2.80GHz',
     speed: 2926,
-    times:
-     { user: 266450,
-       nice: 1480,
-       sys: 34920,
-       idle: 1072572010,
-       irq: 30 } } ]
+    times: {
+      user: 266450,
+      nice: 1480,
+      sys: 34920,
+      idle: 1072572010,
+      irq: 30
+    }
+  }
+]
 ```
 
-Note that since `nice` values are UNIX centric in Windows the `nice` values of
+*Note*: Because `nice` values are UNIX-specific, on Windows the `nice` values of
 all processors are always 0.
 
 ## os.endianness()
 <!-- YAML
 added: v0.9.4
 -->
 
-Returns the endianness of the CPU. Possible values are `'BE'` for big endian
-or `'LE'` for little endian.
+The `os.endianness()` method returns the endianness of the CPU.
```

Comment:
I'm not sure if this one is determined at compile time or not. I think it probably is though and if I am right, this should probably be reworded to mention that it's the endianness of the binary or the endianness of the platform the binary was compiled for (or something to that effect)?